src/main/java/com/teamwizardry/librarianlib/features/particlesystem/modules/SpriteRenderModule.kt

```diff
@@ -64,7 +64,15 @@ class SpriteRenderModule @JvmOverloads constructor(
         /**
          * Whether to enable OpenGL blending
          */
-        @JvmField val enableBlend: Boolean = true
+        @JvmField val enableBlend: Boolean = true,
+        /**
+         * The size of the sprite sheet (must be a power of 2)
+         */
+        @JvmField val spriteSheetSize: Int = 1,
+        /**
+         * The sprite index (indexed left-to-right, top-to-bottom)
+         */
+        @JvmField val spriteIndex: ReadParticleBinding = ConstantBinding(0.0)
```

Comment:
Is this binary compatible?
It might be thanks to `@JvmOverloads` but please check it really is

Reply:
I can't seem to get the binary compat checker running, but from looking at the decompiled source, as far as I can tell, it is.